skimage/transform/_geometric.py

```diff
@@ -1197,6 +1197,12 @@ class EuclideanTransform(ProjectiveTransform):
     translation parameters. The similarity transformation extends the Euclidean
     transformation with a single scaling factor.
 
+    In 2D and 3D, the transformation parameters can be given as the homogeneous
+    transformation matrix, above, or as the implicit parameters, rotation and
+    translation in x (a1) and y (b1). Beyond 3D, translation can be specified
+    using the implicit parameter (again, in x (a1), y (b1), etc.), otherwise,
+    only the matrix form is allowed.
```

Comment:
```suggestion
    translation in x (a1) and y (b1). Beyond 3D, only the translation can be specified
    (again, in x (a1), y (b1), etc.), otherwise, only the matrix form is allowed.
```
I am not sure why `translation` is an implicit parameter?

Reply:
I think implicit parameter is the term used for a method argument (e.g., scale, rotation and translation). It is used in AffineTransform and SimilarityTransform. I'm not sure why they are called implicit either. Parameter seems more accurate. But I'd change it in AffineTransform and SimilarityTransform at the same time.

Reply:
It's not clear to me how the sentence "Beyond 3D, (...), otherwise, (...)" should be parsed: At first look, it seems as if 'otherwise' should encompass all cases which are not *beyond 3D*, i.e., "in 2D and 3D" but... these cases were just covered in the previous sentence. So...?

Reply:
When specifying a similarity transform in more than 3 dimensions involving only translation, the implicit translation parameter can be used. if you need to specify both rotation and translation, then the matrix form must be used. That is based off a similar block of text in the AffineTransform class.

We could say something like, "Beyond 3D, a translation transform can be specified using the implicit parameter (again, in x (a1), y (b1), etc.), but if both translation and rotation are desired, only the matrix form is allowed.

Reply:
@scott-vsi thank you for this response, it's now clear to me. I would suggest writing the docstring along these lines:
"""
In 2D and 3D, the transformation parameters may be passed either through `matrix` (homogeneous transformation matrix above), or through `rotation` and/or `translation` (where the first element of the sequence, `a1`, is the translation along `x`, the second element, `b1`, along `y`, etc.). Beyond 3D, if the transformation is only a translation, you may use `translation`; otherwise, you have to use `matrix`.
"""

Reply:
Ok. Thanks! I updated it. I kept the term implicit parameter because that is the term used throughout this file. If we want to change that I would do it throughout.